content/en/load-testing/Manual/900-create-report-cheatsheet.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Create Report Cheat-Sheet"
+linkTitle: "Cheat-Sheet Create_report"
+
+weight: 900
+type: docs
+
+description: >
+  You will find the usage instructions and a list of available command options below.
+---
+## Usage
+- `create_report [<options>] <inputdir>` is used to generate a test report.
+- `create_report --help` displays all the available command options and a brief guide about their usage.
+
+### Example(Windows PowerShell)
+
+- A report can be generated in the following way, without using any options.
+```powershell 
+C:\xlt-5.7.1\bin> ./create_report.cmd ../results/20991231-112110 
```

Comment:
The rest of the documentation does not specify which OS to use, but examples are as unspecific as possible (see https://xltdoc.xceptance.com/load-testing/quick-start/20-demo-application/), i.e.
* don't put your working path (`C:\xlt-5.7.1\bin`) but use a `cd` command to show where to navigate to run the command, like 
```
cd <XLT>/bin
./create_report.sh
```
* I would prefer if you used a path that makes sense, e.g. `20211231-112110` (like, any valid date in the recent past)